libvirt/resource_libvirt_volume.go

```diff
@@ -254,10 +254,10 @@ func resourceLibvirtVolumeCreate(d *schema.ResourceData, meta interface{}) error
 	if _, ok := d.GetOk("source"); ok {
 		err = img.Import(newCopier(client.libvirt, volume, volumeDef.Capacity.Value), volumeDef)
 		if err != nil {
-		//  don't save volume ID  in case of error. This will taint the volume after.
-		// If we don't throw away the id, we will keep instead a broken volume.
-		// see for reference: https://github.com/dmacvicar/terraform-provider-libvirt/issues/494
-		d.Set("id", "")	
+			//  don't save volume ID  in case of error. This will taint the volume after.
```

Comment:
perhaps we can remove the identation here since not related but ok

Reply:
I tried reverting it back to original. I think the changes were the result of `go-fmt`.